qa/rpc-tests/test_framework/mininode.py

```diff
@@ -623,9 +847,12 @@ def rehash(self):
         return self.sha256
 
     def __repr__(self):
-        return "CBlockHeader(nVersion=%i hashPrevBlock=%064x hashMerkleRoot=%064x nTime=%s nBits=%08x nNonce=%08x)" \
+        aux_pow = ""
+        if self.aux_pow:
+            aux_pow = " aux_pow=%s" % repr(self.aux_pow)
```

Comment:
I think not using the same name would be easier to read here, but no big deal. I did spend 10 seconds extra before I understood, which is why I bothered to comment. :P